keep-ui/app/posthog-client.tsx

```diff
@@ -38,20 +38,17 @@ const PHProvider: React.FC<PHProviderProps> = ({ children }) => {
                   url = url + `?${searchParams.toString()}`;
               }
               const posthog_id = user?.name;
-              console.log("PostHog ID: " + posthog_id);
```

Comment:
why not keeping it?

Reply:
https://github.com/keephq/keep/pull/159#discussion_r1216653092